controller/src/settings/esp.rs

```diff
@@ -244,6 +244,50 @@ impl EspPlayerSettings {
     }
 }
 
+#[derive(Clone, Copy, Deserialize, Serialize, PartialEq, PartialOrd)]
+pub struct EspBombSettings {
+    pub bomb_site: bool,
+    pub bomb_site_color: EspColor,
+
+    pub bomb_position: bool,
+    pub bomb_position_color: EspColor,
+
+    pub bomb_status: bool,
+    pub bomb_status_color: EspColor,
```

Comment:
More reasonable would be `BombEspColor` with the following options:
- Distance
- Time 'till detonation